examples/07-EMIT/ComputeChannelSeparation.py

```diff
@@ -0,0 +1,443 @@
+"""
+EMIT: Compute required channel separation
+-------------------------------------------
+This example shows how you can use PyAEDT to open an AEDT project with
+an EMIT design and analyze the results to determine the required channel
+separation for overlapping bands. 
+"""
+###############################################################################
+# Perform required imports
+# ~~~~~~~~~~~~~~~~~~~~~~~~
+# Perform required imports.
+import os
+import sys
+import time
+import subprocess
+import pyaedt
+from pyaedt import Emit
+from pyaedt.emit import Revision
+from pyaedt.emit import Result
+from pyaedt.modeler.circuits.PrimitivesEmit import EmitComponent
+
+# Check to see which Python libraries have been installed
+reqs = subprocess.check_output([sys.executable, '-m', 'pip', 'freeze'])
+installed_packages = [r.decode().split('==')[0] for r in reqs.split()]
+
+# Install required packages if they are not installed
+def install(package): 
+    subprocess.check_call([sys.executable, '-m', 'pip', 'install', package])
+
+# Install any missing libraries
+required_packages = ['plotly', 'tqdm', 'matplotlib', 'numpy']
+for package in required_packages:
+    if package not in installed_packages:
+        install(package)
+
+# Import required modules
+import plotly.graph_objects as go
+from tqdm.notebook import tqdm
+    
+from matplotlib import pyplot as plt
+plt.ion() # enables interactive mode so plots show immediately
+plt.show()
+import numpy as np
+
+###############################################################################
+# Set non-graphical mode
+# ~~~~~~~~~~~~~~~~~~~~~~
+# Set non-graphical mode. ``"PYAEDT_NON_GRAPHICAL"``` is needed to generate
+# documentation only.
+# You can set ``non_graphical`` either to ``True`` or ``False``.
+# The ``NewThread`` Boolean variable defines whether to create a new instance
+# of AEDT or try to connect to existing instance of it if one is available.
+
+non_graphical = os.getenv("PYAEDT_NON_GRAPHICAL", "False").lower() in ("true", "1", "t")
+NewThread = False
+desktop_version = "2023.2"
+
+###############################################################################
+# Launch AEDT with EMIT
+# ~~~~~~~~~~~~~~~~~~~~~
+# Launch AEDT with EMIT. The ``Desktop`` class initializes AEDT and starts it
+# on the specified version and in the specified graphical mode.
+
+d = pyaedt.launch_desktop(desktop_version, non_graphical, NewThread)
+emitapp = Emit(pyaedt.generate_unique_project_name())
+
+###############################################################################
+# Create and connect EMIT components
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Set up the scenario with radios connected to antennas
+
+def add_and_connect_radio(radio_name, schematic_name=""):
+    """Add a radio from the EMIT library and connect
+    it to an antenna.
+    Returns: 
+        Instance of the radio.
+    Argments:
+        radio_name - String name of the EMIT library radio
```

Comment:
```suggestion
        radio_name: String name of the EMIT library radio
```
Later you used colons rather than a hypen--and I think it is more "readable". Please change any other argument descriptions in the example to follow this convention. Thanks!